deploy/deploy-systemd.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+BRANCH=$1
+name=
+
+if [ "x$BRANCH" = "x" ]; then
+  echo "Please pass branch to deploy as first argument"
+  exit 1
+fi
+
+if [ "x$BRANCH" = "xproduction" ]; then
+  name=foodsaving-world
+else
+  name=foodsaving-world-dev
+fi
+
+function restart-workers() {
+  sudo systemctl restart "${name}-worker.target"
+}
+
+backend_dir="/var/www/${name}/www"
```

Comment:
I remember looking into supervisord some time ago and it looked quite good. But right now I don't see any reason to use it over systemd.
Config has yet another format with similar complexity than systemd service files: https://github.com/celery/celery/blob/master/extra/supervisord/supervisord.conf